### PR TITLE
Rendre visible la timeline des changements d'états des tests ludwig

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to sgmap/ludwig-ui#41

### CODE REVIEW

**Comment by MattiSG** [comment]
GTM

File: lib/models/acceptanceTest.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ var AcceptanceTestSchema = new mongoose.Schema({
     expectedResults: [DroitSchema],
     currentStatus: { type: String, enum: ['accepted-exact', 'accepted-2pct', 'accepted-10pct', 'rejected'] },
     lastExecution: { type: mongoose.Schema.Types.ObjectId, ref: 'AcceptanceTestExecution' },
+    resultUpdated: {type: Date},
```
Comment (MattiSG) [inline lib/models/acceptanceTest.js]: Missing spaces within curly braces for inline objects.

**Comment by MattiSG** [comment]
GTM

File: lib/models/acceptanceTest.js (hunk 1 above)
Comment (fpagnoux) [inline lib/models/acceptanceTest.js]: I'll try to add this one to JSCS

**Pull request merged**